output/v0.0.json

```diff
@@ -3615,6 +3692,23 @@
                     "payment_amount_cents": {
                         "type": "integer",
                         "readOnly": true
+                    },
+                    "metadata": {
+                        "type": "object",
+                        "readOnly": true,
+                        "description": "Additional private data represented as key-value pairs."
+                    },
+                    "reconciliation_id": {
+                        "type": "string",
+                        "readOnly": true
```

Comment:
Description: A string to identify the invoice in your accounting system. Max-length = 50